File: komock-core/src/main/kotlin/ua/com/lavi/komock/engine/handler/after/EmptyAfterResponseHandlerImpl.java
package ua.com.lavi.komock.engine.handler.after;

import org.jetbrains.annotations.NotNull;
import ua.com.lavi.komock.engine.model.Request;
import ua.com.lavi.komock.engine.model.Response;

public class EmptyAfterResponseHandlerImpl implements AfterResponseHandler {
    @Override
    public void handle(@NotNull Request request, @NotNull Response response) {
        // nothing to do
    }
}

File: komock-core/src/main/kotlin/ua/com/lavi/komock/engine/handler/after/EmptyAfterResponseHandlerImpl.kt
package ua.com.lavi.komock.engine.handler.after

import ua.com.lavi.komock.engine.model.Request
import ua.com.lavi.komock.engine.model.Response

/**
 * Created by Oleksandr Loushkin
 */

class EmptyAfterResponseHandlerImpl : AfterResponseHandler {
    override fun handle(request: Request, response: Response) {
        // nothing to do
    }
}
